### ISSUE TITLE
Is PUSH_ACTIVITY  defunct?

### ISSUE THREAD

**Issue by danielbrunt57** [Apollon77/alexa-remote#245]
**Describe the bug**  
![image](https://github.com/user-attachments/assets/3e9a3228-91d3-4809-9aab-ae77958c1461
Is this no longer being pushed by Amazon?

**Comment by Apollon77** [comment]
Yes, since 7.0 release ... so 8 months already? ;-) so newest version tries to simulate it best it can
